[
  {
    "name": "configobj",
    "version": "5.0.6",
    "vulns": [
      {
        "id": "GHSA-c33w-24p9-8m24",
        "fix_versions": [],
        "description": "All versions of the package configobj are vulnerable to Regular Expression Denial of Service (ReDoS) via the validate function, using (.+?)\\((.*)\\). **Note:** This is only exploitable in the case of a developer, putting the offending value in a server side configuration file."
      }
    ]
  },
  {
    "name": "redis",
    "version": "4.4.0",
    "vulns": [
      {
        "id": "GHSA-24wv-mv5m-xv4h",
        "fix_versions": [
          "4.3.6",
          "4.4.3",
          "4.5.3"
        ],
        "description": "redis-py before 4.5.3, as used in ChatGPT and other products, leaves a connection open after canceling an async Redis command at an inopportune time (in the case of a pipeline operation), and can send response data to the client of an unrelated request in an off-by-one manner. The fixed versions for this CVE Record are 4.3.6, 4.4.3, and 4.5.3, but [are believed to be incomplete](https://github.com/redis/redis-py/issues/2665). CVE-2023-28859 has been assigned the issues caused by the incomplete fixes."
      },
      {
        "id": "GHSA-8fww-64cx-x8p5",
        "fix_versions": [
          "4.4.4",
          "4.5.4"
        ],
        "description": "redis-py through 4.5.3 and 4.4.3 leaves a connection open after canceling an async Redis command at an inopportune time (in the case of a non-pipeline operation), and can send response data to the client of an unrelated request. NOTE: this issue exists because of an incomplete fix for CVE-2023-28858."
      }
    ]
  }
]